gulpfile.babel.js

```diff
@@ -265,7 +265,7 @@ gulp.task('scripts', ['jscs', 'jshint'], () => {
 });
 
 // Clean Output Directory
-gulp.task('clean', del.bind(null, ['dist', '.publish'], {dot: true}));
+gulp.task('clean', () => del(['dist', '.publish']));
```

Comment:
The existing usage was incorrect as it never called the callback, so gulp didn't know when it ended. With del@2 it's a promise so we can just return it. Also removed the `dot: true` option as it's only needed to match dotfiles when using globbing.